sdk/monitor/azure-monitor-opentelemetry/samples/tracing/django/sample/example/views.py

```diff
@@ -3,12 +3,14 @@
 # Licensed under the MIT License. See License in the project root for
 # license information.
 # --------------------------------------------------------------------------
-
+import logging
 from django.http import HttpResponse
 
+logger = logging.getLogger("app_logger")
```

Comment:
I think if we are not using the root logger, we should at least use a valid module to show Cxs how to instrument a logging at the module level for their app.